src/ace/managers/viewport/scrollbuffer.c

```diff
@@ -75,22 +75,42 @@ tScrollBufferManager *scrollBufferCreate(void *pTags, ...) {
 	// Create copperlist entries
 	pCopList = pVPort->pView->pCopList;
 	if(pCopList->ubMode == COPPER_MODE_BLOCK) {
-		pManager->pStartBlock = copBlockCreate(
+		pManager->copStart.pStartBlock = copBlockCreate(
```

Comment:
those can be anonymous unions to skip the copStart/copBreak stuff, or if not, union contents can have shorter names to not duplicate the "start/break" and "copper" info from union name.

Something like `pManager->copStart.pBlock` and `pManager->copStart.uwOffset`

Reply:
Made it anonymous